docs/sources/alerting/fundamentals/_index.md

```diff
@@ -91,40 +95,40 @@ A rule using the PromQL expression above creates as many alert instances as the
 
 {{< figure src="/static/img/docs/alerting/unified/multi-dimensional-alert.png" caption="Multiple alert instances from a single alert rule" >}}
 
-[Alert rules are frequently evaluated](ref:alert-rule-evaluation) and the state of their alert instances is updated accordingly. Only alert instances that are in a firing or resolved state are routed to notification policies to be handled.
+[Alert rules are frequently evaluated](ref:alert-rule-evaluation) and the state of their alert instances is updated accordingly. Only alert instances that are in a firing or resolved state are sent for notifications.
```

Comment:
This one is tricky. 

This may be interpreted that an alert instance is sent "as a notification". 

But this is not how it works. 

An alert instance will then be included within an alert group, and a notification will be sent for the entire group. 

If you think, `as` is still correct, please lmk and will accept the suggestion.

Reply:
how about "in" then?

Reply:
fixed by https://github.com/grafana/grafana/pull/89155/commits/e910891e559e5125b479617d839371e98045e7c8